src/core/particle_data.cpp

```diff
@@ -2304,8 +2295,3 @@ bool particle_exists(int part) {
         return true;
    return false;
 }
-
-#ifndef MASS
-constexpr double ParticleProperties::mass;
```

Comment:
Because the standard says so (9.4.2): "A static data member of literal type can be declared in the class definition with the constexpr specifier; if so, its declaration shall specify a brace-or-equal-initializer in which every initializer-clause that is an assignment-expression is a constant expression. [ Note: In both these cases, the member may appear in constant expressions. — end note ] The member shall still be defined in a namespace scope if it is odr-used (3.2) in the program and the namespace scope definition shall not contain an initializer."